src/autorun.ts

```diff
@@ -67,20 +71,37 @@ export const _ = forwardTracker('_');
 export const computed = <T>(fn: Cb<T>): Observable<T> => new Observable(observer => {
     const deps = new Map<Observable<unknown>, TrackEntry<unknown>>();
     const update$ = new Subject<Update>();
-    const createTrackers = (track: boolean) => {
-        const r  = createTracker(track, Strength.Normal) as $FnWithTrackers;
-        r.weak   = createTracker(track, Strength.Weak);
-        r.normal = createTracker(track, Strength.Normal);
-        r.strong = createTracker(track, Strength.Strong);
-        return r;
+    const error$ = new Subject<void>();
```

Comment:
I think `error$` thread could be dropped and we could use `observer.error` directly. Though it seems a bit weird. I decided to keep a separate Subject for now, till we decide with in-place error throwing #29